docs/man/bicker_ser.txt

```diff
@@ -34,6 +34,60 @@ EXTRA ARGUMENTS
 
 This driver supports no extra arguments from linkman:ups.conf[5].
 
+VARIABLES
+---------
+
+Depending on the type of your UPS unit, some of the following variables may
+be changed with linkman:upsrw[8]. If the driver can't read a variable from the
+UPS, it will not be made available. Whenever not explicitly stated, any variable
+can be disabled, in which case the action it performs will not be executed. To
+disable a variable, set it to an empty value.
+
+*output.current.low* (in mA, default `200`)::
+If activated and the UPS is in battery mode and the current drops below the set
+value, the output of the UPS will shut down and disconnect the energy storage to
+prevent self-discharge.
+
+*ups.delay.shutdown* (in seconds, default disabled)::
+If activated and the UPS is in battery mode and the set time has expired, the
+output will be disabled, and the UPS and energy storage will be disconnected.
+
+*ups.delay.shutdown.signal* (in seconds, default disabled)::
+If activated and the UPS is in battery mode and the set time has elapsed, a
+shutdown command via USB or relay (relay event) is signaled.
+
+*ups.delay.start* (in seconds, default disabled)::
+If activated and a restart condition switches the UPS output off and on again,
+the set time is the delay between switching on and off. The time should cause a
+defined off time so that capacities in the application can be discharged.
+
+*battery.charge.restart* (in percent, default disabled)::
+If activated and the UPS is off or restarts, the UPS output will not be released
+until the energy storage device has the set charge state. The energy storage
+device is charged in the meantime.
+
+*ups.delay.shutdown.in1* (in seconds, default disabled)::
+If activated and the UPS is in battery mode and the signal at the IN-1 input is
+high and the set time has expired, a shutdown command via relay event is
+signaled.
+
+*battery.charge.low* (in percent, default `20`)::
+If activated and the UPS is in battery mode and the battery level drops below
+the set value, a shutdown command via relay event is signaled.
+
+*battery.charge.low.empty* (in percent, default `20`)::
+This parameter stores the threshold value for the "Battery Empty" signal.
+Currently this setting is only valid for relay signaling. Cannot be disabled.
+
+*ups.relay.mode* (default `0x01`)::
+This parameter controls the behavior of the relay in case of different events.
+Cannot be disabled.
+- `0x01` On power fail (normally closed)
+- `0x02` On power fail (normally opened)
+- `0x03` Shutdown impulse (1 second)
+- `0x04` Battery low signal (normally closed)
+- `0x05` Battery defect signal (normally closed)
+
```

Comment:
Can you please check this block in man/html rendition? I am afraid `a2x` would not make a bulleted list of these lines, but would place them into one long line. It may need a blank before start of list (or do I confuse that bit with markdown?) In case of "blank line as part of block", to keep the indentation, a single "plus" character on the blank line is used.

Reply:
According to [asciidoctor](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#mixing-lists):

> Description lists are quite lenient about whitespace [...]

so it shoud be fine with or without extra lines. Anyway I ended up converting that into a nested (horizontal) description list in 9c3dedff16d96cbc79e26e839a9747a0dec51241 as it seems more appropriate.